configs/openSUSE/security.toml

```diff
@@ -26,15 +26,29 @@ Locations = [
     "/usr/share/polkit-1/rules.d/"
 ]
 
-[FileDigestGroup.atop."boo#12345"]
+[FileDigestGroup.file-signature."boo#12345"]
 note = """
 Builds statistics based on Squid logfile metadata.
 Include sarg-reports which is SUSE specific and important for privilege dropping.
 """
 
-[FileDigestGroup.atop."boo#12345".digests]
-"/etc/cron.daily/suse.de-sarg" = "d536dc68e198189149048a907ea6d56a7ee9fc732ae8fec5a4072ad06640e359"
-"/etc/cron.monthly/suse.de-sarg" = "d536dc68e198189149048a907ea6d56a7ee9fc732ae8fec5a4072ad06640e359"
+[FileDigestGroup.file-signature."boo#12345".digests."/etc/cron.daily/suse.de-sarg"]
```

Comment:
The configuration format has become quite verbose and redundant now, because we have to repeat the `FileDigestGroup.file-signature."<bug>".digests.` prefix for each path. Items belonging together aren't structured together any more. This could be solved by adding some "ascii art" like comments or whitespace in the file. Still it feels harder to read now.

Reply:
Should be much better now. Is it fine now?

Reply:
Also here I can't see a difference. Am I looking at an oudated version or did you forget to commit something?

Reply:
Now please take a look at the changes for `test.config` where you'll see the new format.